docs/reference/relnotes/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Knative Release Notes"
+linkTitle: "Release notes"
+weight: 10
+type: "docs"
+---
+
+- For details about the latest Knative Serving release, see the [https://github.com/knative/serving/releases](Knative Serving releases page).
```

Comment:
```suggestion
- For details about the latest Knative Serving release, see the [https://github.com/knative/serving/releases] (Knative Serving releases page).
```

Maybe needs a space? It's continuing the link which is what is what is causing failed tests.

Reply:
It shouldn't, this is standard md syntax? https://www.markdownguide.org/basic-syntax/#links

Reply:
Ah maybe you just have it reversed. 

```suggestion
- For details about the latest Knative Serving release, see the [Knative Serving releases page](https://github.com/knative/serving/releases).
```

https://github.com/knative/docs/blob/b58a19729113be02200cc2e7188440e257416d21/docs/reference/relnotes/_index.md <--- you can see that it's including Knative

Reply:
LMAO this is exactly why I am having more coffee right now, derp. TGIF and thanks for catching this, I'll update it here in a bit :slightly_smiling_face:

Reply:
Not a problem, I'm a bit out of it today too - it's basically Sepia outside. I had to look a couple times to make sure I wasn't missing something.